### PR TITLE
Add learning rate scheduler

### PR DESCRIPTION
**Pull request by klieret**

### CODE REVIEW

**Comment by GageDeZoort** [comment]
Thanks for adding, let's make `step_size` and `gamma` hyperparameters for the trainer.

**Comment by klieret** [comment]
I think the parameter on the specific scheduler though, right? I think it's better to keep it like this and use it as follows:

```python
scheduler = partial(ExponentialLR, gamma=0.9)
trainer = GraphTCNTrainer(lr_scheduler=scheduler, ...)
```
